src/__tests__/applicationForm.int.test.ts

```diff
@@ -0,0 +1,70 @@
+import request from 'supertest';
+import { app } from '../app';
+import { db } from '../database';
+import type { Result } from 'tinypg';
+
+const agent = request.agent(app);
+
+describe('/applicationForms', () => {
+  describe('GET /', () => {
+    it('returns an empty array when no data is present', async () => {
+      await agent
+        .get('/applicationForms')
+        .expect(200, []);
+    });
+
+    it('returns all application forms present in the database', async () => {
+      await db.query(`
+        INSERT INTO opportunities (
+          title,
+          created_at
+        )
+        VALUES
+          ( 'Tremendous opportunity 👌', '2525-01-01T00:00:05Z' );
+      `);
+      await db.query(`
```

Comment:
This test was half baked.  It passes but only because it isn't actually testing the right endpoint.